packages/theme/components/WishlistSidebar.vue

```diff
@@ -38,6 +38,7 @@
                     <SfProperty v-for="(attribute, key) in wishlistGetters.getItemAttributes(product, ['color', 'size'])" :key="key" :name="key" :value="attribute"/>
                   </div>
                 </template>
+                <template #actions>{{ }}</template>
```

Comment:
Is that needed?

Reply:
if we want to hide the "save for later" button yes, sfcollectedProduct has build-in slot "actions" with a  button that has no functionality. We have to overwrite it to hide it

Reply:
Ah :) Do we need the curly braces? Or would `<template #actions />` or `<template #actions></template> be enough? Not sure if we need to interpolate an empty string inside

Reply:
iirc we do need them, but i'm not 100% sure, will test it